public/catch-js-errors.js

```diff
@@ -89,23 +92,32 @@
 	}
 
 	if ( isLocalStorageNameSupported() ) {
+		assignment = Math.random();
 		// Randomly assign 1% of users to log errors
-		if ( localStorage.getItem( 'log-errors' ) === undefined ) {
-			if ( Math.random() <= 0.01 ) {
-				localStorage.setItem( 'log-errors', 'true' );
+		if ( ! localStorage.getItem( 'log-errors' ) ) {
+			if ( assignment <= 0.01 ) {
+				localStorage.setItem( 'log-errors', 'atlas' );
+			} else if ( assignment <= 0.02 ) {
+				localStorage.setItem( 'log-errors', 'analytics' );
+				//Prep the stage up for GA logging experiment
+			} else if ( assignment <= 0.03 ) {
+				localStorage.setItem( 'log-errors', 'external' );
+				//Prep the stage up for Google Stackdriver or other service experiment
 			} else {
 				localStorage.setItem( 'log-errors', 'false' );
 			}
 		}
 
-		if ( localStorage.getItem( 'log-errors' ) !== undefined && localStorage.getItem( 'log-errors' ) === 'true' ) {
+		if ( localStorage.getItem( 'log-errors' ) !== undefined &&
```

Comment:
We should just set it to a constant first and then do the conditionals against the constant, versus hitting localStorage for each check.